test/e2e/specs/site-editor/title.spec.js

```diff
@@ -26,7 +26,7 @@ test.describe( 'Site editor title', () => {
 			'role=region[name="Editor top bar"i] >> role=heading[level=1]'
 		);
 
-		await expect( title ).toHaveText( 'Editing template: Index' );
+		await expect( title ).toHaveText( 'Editing template:Index' );
```

Comment:
I think there's no need for space here. a11y tree renders text correctly, and I re-tested with VoiceOver, which also reads it correctly.